corehq/apps/style/static/style/less/bootstrap3/report.less

```diff
@@ -15,8 +15,8 @@
   .border-top-radius(5px);
   .border-bottom-radius(5px);
   left: 50%;
-  width: 150px;
-  margin-left: -75px;
+  width: 350px;
+  margin-left: -175px;
```

Comment:
negative margins i feel like are only used in weird circumstances and it's usually not clear what. i'm not certain what a `-175px` margin is doing on an element